schemas/iso19139/src/main/plugin/iso19139/present/csw/iso-summary.xsl

```diff
@@ -21,6 +21,7 @@
 			<xsl:apply-templates select="gmd:parentIdentifier"/>
 			<xsl:apply-templates select="gmd:hierarchyLevel"/>
 			<xsl:apply-templates select="gmd:hierarchyLevelName"/>
+			<gmd:contact />
```

Comment:
I don't know if we should add empty elements not requested by the CSW output standard to only make the output document valid for the XSD or the schematron ...

If we really want, 
* In that case the brief output should be also updated ?
* why not adding the value from the record if present and not always add an empty contact or scope ? - but then it does not match what's in CSW 2.0.2 ?